neqo-bin/src/udp.rs

```diff
@@ -28,6 +28,13 @@ impl Socket {
     pub fn bind<A: std::net::ToSocketAddrs>(addr: A) -> Result<Self, io::Error> {
         let socket = std::net::UdpSocket::bind(addr)?;
 
+        // Try to incresase the send and receive buffer sizes.
+        let socket = socket2::Socket::from(socket);
+        let one_mb: usize = 1 << 20;
+        // socket.set_send_buffer_size(one_mb)?;
+        socket.set_recv_buffer_size(one_mb)?;
```

Comment:
For the record, we use a 1MiB buffer in Firefox as well:

https://searchfox.org/mozilla-central/rev/9c395b6371eaea0d15f9c8a37889022be350cf0b/modules/libpref/init/StaticPrefList.yaml#14560-14564

https://searchfox.org/mozilla-central/rev/01aaa47e62a2015e7641f26ab0bc2bb00ab579b8/netwerk/protocol/http/HttpConnectionUDP.cpp#107-115